include/psen_scan_v2/laserscan_conversions.h

```diff
@@ -22,6 +22,14 @@
 
 namespace psen_scan_v2
 {
+/**
+ * @brief Converts the technical monitoring_frame to the user friendly LaserScan type sent by the
+ * IScanner::LaserScanCallback.
+ * @see LaserScan
+ * @see monitoring_frame::Message
+ * @see ScannerV2
+ * @see IScanner
```

Comment:
```suggestion
 * @see monitoring_frame::Message
 * @see ScannerV2
```
those are already linked anyways